core/modules/config/tests/config.test

```diff
@@ -158,6 +158,27 @@ class ConfigurationTest extends BackdropWebTestCase {
     $this->assertEqual(FALSE, $config_data->isOverridden('foo.key1'), 'Original value no longer overridden.');
     $this->assertEqual($config_data->get('foo.key1'), 'chocolate', 'Original value has been updated to include overridden value.');
   }
+
+  /**
+   * Tests that config directories are protected by .htaccess files.
+   */
+  public function testConfigHtaccess() {
+    // Check that the .htacess file is present.
+    $active_dir = config_get_config_directory();
+    $this->assertTrue(is_file($active_dir . '/.htaccess'), "Successfully located the .htaccess file in the 'active' config directory.");
+
+    // Verify contents of .htaccess file.
+    $file = file_get_contents($active_dir . '/.htaccess');
+    $this->assertEqual($file, file_htaccess_lines(FALSE), 'The .htaccess file contains the proper content.');
```

Comment:
file_htaccess_lines(FALSE) - would create a file for public directories. You want private ones.

`file_htaccess_lines()`

Reply:
thanks!